scap/src/capturer/engine/win/mod.rs

```diff
@@ -66,29 +139,115 @@ impl WindowsCaptureHandler for Capturer {
 
 impl WinStream {
     pub fn start_capture(&mut self) {
-        // TODO: Prevent cloning the transmitter
+
         let capture_control = Capturer::start_free_threaded(self.settings.clone()).unwrap();
         self.capture_control = Some(capture_control);
     }
 
     pub fn stop_capture(&mut self) {
-        let mut capture_control = self.capture_control.take().unwrap();
-        capture_control.stop();
+        let capture_control = self.capture_control.take().unwrap();
+        let _ = capture_control.stop();
     }
 }
 
-pub fn create_capturer(options: &Options, tx: mpsc::Sender<Frame>) -> WinStream {
+#[derive(Clone, Debug)]
+struct FlagStruct {
+    pub tx: mpsc::Sender<Frame>,
+    pub crop: Option<CGRect>,
+}
+
+pub fn create_capturer(
+    options: &Options,
+    tx: mpsc::Sender<Frame>,
+) -> WinStream {
     let settings = Settings::new(
         Monitor::primary().unwrap(),
         Some(true),
         None,
-        ColorFormat::Rgba8,
-        tx,
-    )
-    .unwrap();
+        ColorFormat::Bgra8,
```

Comment:
Let's get this color format from the frameType the user passes in options.

Reply:
We can match for `optioms.FrameType` and return Bgra8 if it's BGRAFrame and Rgba8 if it's RGBAFrame.

Does that make sense?

Reply:
Modifying the struct

Reply:
Will need some help here @Pranav2612000